doc/_static/custom_styles.css

```
@@ -15,14 +15,18 @@ div.fasb h3 {
   display: inline-block;
 }
 
+/* Custom bottom margin for the Where to go now section */
+#where-to-go-now {
+	margin-bottom: 18px;
+}
+
```

```suggestion
/* Standard bottom margin for the card sections */
.section .sphinx-bs {
  margin-bottom: 30px;
}

```

Instead of tying the rule to this one particular section title, it makes more sense and is less fragile and bespoke if we instead actually solve the root problem, the cards element not having enough bottom margin, unlike the other elements.

IO we were to take a bespoke approach, it should be simply adding a line break (`|`) in the rst source file, which won't break if the section name changes and add another whole custom CSS selector.

---

Also, fixed a hard tab in the file, inconsistent with the rest of the formatting.

---

Why 30px? That's too really much space.

---

Its the standard margin everything else uses, plus the existing padding on the element. I don't think its excessive, but if you'd prefer it with just the standard margin and not the extra padding, the following will accomplish that:

```suggestion
/* Standard bottom margin for the card sections */
.section .sphinx-bs {
  margin-bottom: 30px;
  padding-bottom: 0 !important;
}

```

---

Sure, but please take a look at the screenshots I added in the description. I carefully selected 18px because that's the amount that looks about the same between the paragraph before "Where to go " and that header, and between the cards and "Join our community".

I don't think this needs to be about using the same amount as everywhere else, just making things look uniform.

---

Okay, I understand your point. I've modified the below to add the extra 4 px to be identical in appearance to your original (1 em (16 px) padding + 18 px margin = 34 px), just organized in a more coherent manner. As I mentioned before, I agree extra padding is justifiable to account for the shadow falloff, which makes the element appear larger than its nominal bounding box, and I should have just used values that would be precisely equivalent to what you had originally—my mistake.

```suggestion
/* Standard bottom margin for the card sections */
.section .sphinx-bs {
  margin-bottom: 30px;
  padding-bottom: 4px !important;
}

```

---

> Okay, thanks. I've modified the below to add the extra 4 px to be identical in appearance to your original (1 em (16 px) padding + 18 px margin = 34 px)

@CAM-Gerlach, this is your way of doing this change, which is exactly like mine (as you're clearly saying above). Therefore, I'm only going to take your suggestion of using `.section .sphinx-bs` so things don't depend on id of this section.

---

The rendered result is the same, but the implementation is more logical, because it applies a consistent, standard margin, plus the extra padding necessary for the particular element, rather than simply applying a bespoke margin on top of a bespoke padding.

---

The commit also didn't pick up my changes to the comment so its not out of sync with the selector. I'll fix that part in #148  .

---

Yeah, I missed the change to the comment, sorry.